src/styles/ChatBubble.css

```
@@ -0,0 +1,118 @@
+.chat-bubble {
+  position: relative;
+  max-width: 260px;
+  padding: 14px 16px;
+  border-radius: 20px;
+  font-size: 15px;
+  line-height: 1.6;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)